storage/innobase/log/log0recv.cc

```diff
@@ -96,6 +96,12 @@ otherwise.  Note that this is false while a background thread is
 rolling back incomplete transactions. */
 volatile bool recv_recovery_on;
 volatile lsn_t backup_redo_log_flushed_lsn;
+extern bool opt_page_tracking;
+extern char *xtrabackup_incremental;
+extern lsn_t incremental_start_checkpoint_lsn;
```

Comment:
Where are we initializing those variables? shouldn't those be part of xb0xb.h?

Reply:
we are intializing them in xtrabackup.cc. All related variables are initialise
>>lsn_t incremental_lsn;
>>lsn_t incremental_start_checkpoint_lsn;
  lsn_t incremental_to_lsn;
  lsn_t incremental_last_lsn;
  lsn_t incremental_flushed_lsn;

Reply:
@rahulmalik87 this is still wrong to me. `xb0xb.h` is the place where we should be externing PXB variables used on shared (pxb / mysql) code.